src/providers/project/python.ts

```diff
@@ -0,0 +1,44 @@
+
+/* IMPORT */
+
+import * as path from 'path';
+import Utils from '../../utils';
+
+/* PYTHON */
+
+async function python ( rootPath = Utils.folder.getActiveRootPath (), ...args ) {
+
+  if ( !await Utils.folder.is ( rootPath ) ) return;
+
+  const pkgPath = path.join ( rootPath, 'package.json' ),
```

Comment:
`package.json` is a file used in the JavaScript ecosystem, I don't think this code is correct for debugging whole Python projects. I'm not sure if there are any conventions analogous to that of the JS ecosystem or what they are. If you're not sure about what to write in this file maybe just remove it and let's only add support for debugging single Python files for now.

Reply:
sounds good. updated.

Reply:
The `project/python.ts` file seems useless now, can you delete it?

Reply:
Thanks for catching.